ckanext/recombinant/logic.py

```diff
@@ -259,7 +259,8 @@ def _dataset_fields(geno):
     """
     return the dataset metadata fields created for dataset definition geno
     """
-    return {'title': geno['title'], 'notes': geno.get('notes', '')}
+    notes = geno.get('alt_notes', geno.get('notes', ''))
+    return {'title': geno['title'], 'notes': notes}
```

Comment:
In gitlib issue 7, they want `replace the record description in the search pages`.